src/Aspire.Cli/Utils/ConsoleActivityLogger.cs

```diff
@@ -18,6 +17,7 @@ namespace Aspire.Cli.Utils;
 internal sealed class ConsoleActivityLogger
```

Comment:
We should actually move a lot of this logic into the interaction service (not this PR).